client/delegate-auth/delegate-auth-script.ts

```diff
@@ -0,0 +1,160 @@
+import {
+    IAM,
+    Encoding,
+    PubKeyType,
+    Algorithms,
+    DIDAttribute,
+    setChainConfig,
+    WalletProvider,
+    setCacheClientOptions,
+} from 'iam-client-lib';
+import axios from 'axios';
+import * as emoji from 'node-emoji';
+import { Keys } from '@ew-did-registry/keys';
+import { providers, Wallet } from 'ethers';
+import { JWT } from '@ew-did-registry/jwt';
+
+const backendUrl = 'http://localhost:3333';
+
+const connectIAM = async (privateKey: string): Promise<IAM> => {
+
+    let connectionInfos = undefined;
+
+    const iamAgent: IAM = new IAM({
+        rpcUrl: "https://volta-rpc.energyweb.org",
+        privateKey,
+    });
+    setCacheClientOptions(73799, {
+        url: "https://identitycache-dev.energyweb.org/v1",
+    });
+
+    setChainConfig(73799, {
+        rpcUrl: "https://volta-rpc.energyweb.org"
+    });
+    connectionInfos = await iamAgent.initializeConnection({
```

Comment:
```suggestion
    const connectionInfos = await iamAgent.initializeConnection({
```
Why not initialize with `const` here?

Reply:
Forgot to update. Initially I made this variable global to the whole script, but then I changed it to be local to this function. `const` is indeed suitable. Thanks @jrhender